addons/survey/views/survey_templates_statistics.xml

```diff
@@ -27,6 +27,7 @@
     </template>
 
     <template id="survey_page_statistics_inner" name="Survey: result statistics content">
+        <t t-set="disabledQuestionsFilters" t-value="{filter['question_id'] for filter in search_filters}"/>
```

Comment:
Let's not call it disabled then, maybe "already_filtered_questions" ? We use it to disable things but it's not what it contains.